ci-operator/step-registry/ipi/conf/aws/blackholenetwork/ipi-conf-aws-blackholenetwork-ref.yaml

```diff
@@ -11,3 +11,8 @@ ref:
       memory: 100Mi
   documentation: |-
     The IPI AWS blackholenetwork configure step generates the AWS-specific install-config.yaml contents based on the cluster profile and optional input files using subnets where the private ones do not have direct egress access.
+  env:
+  - name: ZONES_COUNT
+    default: "2"
+    documentation: |-
+      The number of AZs to present in the cluster. Accepted values are 1 and 3.
```

Comment:
nit: 2 is the default, so it's not that likely that callers would set it explicitly.  But it's probably worth including it in the set of acceptable values anyway, like `Accepted values are 1, 2, and 3.`.